request-signing-v2.md

```diff
@@ -17,16 +17,16 @@ You can generate a key pair using [`openssl`](https://www.openssl.org/).
 To generate the private key, run:
 
 ```bash
-docker run --rm -v ${PWD}:/out -w /out -it alpine/openssl ecparam -genkey -name secp521r1 -noout -out ec512-private-key.pem
+openssl ecparam -genkey -name secp521r1 -noout -out ec512-private-key.pem
```

Comment:
This could potentially cause issues as different systems might use a different version of OpenSSL, whereas using Docker guarantees some consistency. I'm curious to know what feedback we got around this exactly.

Reply:
Made a change to follow approach proposed above.

Reply:
I think it's great to list both options indeed. Thank you!